profiler/tests/cell_label_harmonization_scanvi_pan_tissue.py

```diff
@@ -0,0 +1,121 @@
+import cellxgene_census
+import numpy as np
+import pandas as pd
+
+
+def get_census_data(
```

Comment:
It doesn't seem right to add a Census-centric test to this repo. I would remove this. (Census could run such a test on its own, using this profiler.)